config/config.go

```diff
@@ -604,10 +607,10 @@ func DefaultP2PConfig() *P2PConfig {
 		MaxNumInboundPeers:           40,
 		MaxNumOutboundPeers:          10,
 		PersistentPeersMaxDialPeriod: 0 * time.Second,
-		FlushThrottleTimeout:         100 * time.Millisecond,
-		MaxPacketMsgPayloadSize:      1024,    // 1 kB
-		SendRate:                     5120000, // 5 mB/s
```

Comment:
[no change needed] it's a bit strange that this says 5120000 == 5 MB/s because as far as I can tell:

5 MB = 5_000_000 bytes
5 MiB = 5_242_880 bytes

and neither of those values match the one used. Definitely no change needed b/c this was the value used before + after this PR

Reply:
I interpreted the 5MB/s in the comment as an attempt to provide an approximate value of the SendRate in a larger unit. However, I concur that it is not accurate.

Reply:
Good point, it could be an approximation so `~5 MB/s`